contracts/TracerPerpetualSwaps.sol

```diff
@@ -204,24 +209,44 @@ contract TracerPerpetualSwaps is
         Balances.Account storage account1 = balances[order1.maker];
         Balances.Account storage account2 = balances[order2.maker];
 
+        int256 fee =
+            PRBMathUD60x18
+                .mul(uint256(quoteChange), uint256(feeRate.toInt256()))
```

Comment:
Shouldn't need to cast `feeRate` to a int256 here

Reply:
You are right, am a silly goose